lib/vsc/install/shared_setup.py

```diff
@@ -1353,9 +1358,6 @@ def action_target(package, *args, **kwargs):
     """
     This main is the setup.py for vsc-install
     """
-    install_requires = [
```

Comment:
i wouldn't remove it, as we might start to add minimal version requirements, and then this becomes useful. otoh, if we would need them, we could readd them. but this basically means: we support whatever el6 ships.